crates/uv-python/src/sysconfig/parser.rs

```diff
@@ -166,18 +166,45 @@ fn parse_string(cursor: &mut Cursor, quote: char) -> Result<String, Error> {
         };
         match c {
             '\\' => {
-                // Handle escaped quotes.
-                if cursor.first() == quote {
-                    // Consume the backslash.
-                    cursor.bump();
-                    result.push(quote);
-                    continue;
-                }
+                // Treat the next character as a literal.
+                //
+                // See: https://github.com/astral-sh/ruff/blob/d47fba1e4aeeb18085900dfbbcd187e90d536913/crates/ruff_python_parser/src/string.rs#L194
+                let Some(c) = cursor.bump() else {
+                    return Err(Error::UnexpectedEof);
+                };
+
+                let c = match c {
+                    '\\' => '\\',
+                    '\'' => '\'',
+                    '\"' => '"',
+                    'a' => '\x07',
+                    'b' => '\x08',
+                    'f' => '\x0c',
+                    'n' => '\n',
+                    'r' => '\r',
+                    't' => '\t',
+                    'v' => '\x0b',
+                    // Special cases where the escape sequence is not a single character
+                    '\n' => {
+                        // Both the backslash and newline are ignored.
+                        continue;
+                    }
+                    '\r' => {
+                        // Both the backslash and newline are ignored.
+                        if cursor.first() == '\n' {
+                            cursor.bump();
+                        }
+                        continue;
+                    }
+                    _ => {
+                        // We don't support octals (e.g., `\ooo`), hexadecimals (e.g., `\xhh`), and
+                        // named Unicode characters (e.g., `\N{...}`).
+                        warn!("Unrecognized escape sequence: `{c}`");
+                        c
```

Comment:
I don't think it's necessary to add parsing for these... We know this file doesn't contain these, we generate it.